include/openenclave/corelibc/wchar.h

```diff
@@ -1,2 +1,11 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
+
+#ifndef _OE_WCHAR_H
+#define _OE_WCHAR_H
+
+#include <openenclave/bits/types.h>
+
+size_t oe_wcslen(const wchar_t* s);
```

Comment:
This header and any other published header would likely need OE_EXTERNC_BEGIN, OE_EXTERNC_END closures to that they can be included in C++ files.